boards/adi/max32660evsys/doc/index.rst

```diff
@@ -0,0 +1,121 @@
+.. zephyr:board:: max32660evsys
+
+Overview
+********
+The MAX32660 evaluation system offers a compact development platform that
+provides access to all the features of the MAX32660 in a tiny, easy to
+use board. A MAX32625PICO-based debug adapter comes attached to the main
+board. It can be snapped free when programming is complete. The debug
+module supports an optional 10-pin Arm® Cortex® debug connector for DAPLink
+functionality. Combined measurements are 0.65in x 2.2in, while the main board
+alone measures 0.65in x 0.95in. External connections terminate in a dual-row
+header footprint compatible with both thru-hole and SMT applications. This
+board provides a powerful processing subsystem in a very small space that
+can be easily integrated into a variety of applications.
+
+The Zephyr port is running on the MAX32660 MCU.
+
+.. image:: img/max32660evsys.webp
+   :align: center
+   :alt: MAX32660 EVSYS
+
+Hardware
+********
+
+- MAX32660 MCU:
+
+  - High-Efficiency Microcontroller for Wearable Devices
+     - Internal Oscillator Operates Up to 96MHz
```

Comment:
```suggestion

     - Internal Oscillator Operates Up to 96MHz
```
 
Ditto in the rest of the list
See https://builds.zephyrproject.io/zephyr/pr/85733/docs/boards/adi/max32660evsys/doc/index.html